src/js/controller/tracks-loader.ts

```diff
@@ -64,14 +80,15 @@ function xhrSuccess(xhr, track, successHandler, errorHandler) {
             const responseText = xhr.responseText;
             if (responseText.indexOf('WEBVTT') >= 0) {
                 // make VTTCues from VTT track
-                loadVttParser().then(VTTParser => {
-                    const parser = new VTTParser(window);
+                loadVttParser().then((VttParser: (window: Window, decoder?: () => void) => void): void => {
+                
+                    const parser = new VttParser(window);
```

Comment:
This isn't being typed a `VTTParser`. The point of importing it is to get the type safety for it here.